### PR TITLE
[13.0][MIG] hr_holidays

### PR DESCRIPTION
**Pull request by Yasaie**
Migration of `hr_holidays`
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_holidays/migrations/13.0.1.5/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        table='hr_leave')
+
+
+def fill_leave_allocation_allocation_type(cr):
+    openupgrade.logged_query(
+        cr, """
+        UPDATE hr_leave_allocation hla
+        SET allocation_type = 'regular'
+        WHERE hla.allocation_type IS NULL
+        """
```
Comment (MiquelRForgeFlow) [inline addons/hr_holidays/migrations/13.0.1.5/post-migration.py]: = 'accrual'
where hr.{} = TRUE
.format(openupgrade.get_legacy_name('accrual'))

File: addons/hr_holidays/migrations/13.0.1.5/openupgrade_analysis_work.txt [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# DONE: post-migration: mapped values
+hr_holidays  / hr.leave.allocation      / accrual (boolean)             : DEL
+hr_holidays  / hr.leave.allocation      / manager_id (many2one)         : NEW relation: hr.employee
+# NOTHING TO DO
+hr_holidays  / hr.leave.allocation      / allocation_type (selection)   : NEW required, selection_keys: ['accrual', 'regular'], req_default: function, hasdefault
+# DONE: post-migration: filled the field when empty
+hr_holidays  / hr.leave.type            / allocation_notif_subtype_id (many2one): NEW relation: mail.message.subtype, hasdefault
+hr_holidays  / hr.leave.type            / categ_id (many2one)           : DEL relation: calendar.event.type
+hr_holidays  / hr.leave.type            / code (char)                   : NEW
+hr_holidays  / hr.leave.type            / create_calendar_meeting (boolean): NEW hasdefault
```
Comment (MiquelRForgeFlow) [inline addons/hr_holidays/migrations/13.0.1.5/openupgrade_analysis_work.txt:23]: post-migration: create_calendar_meeting = False if categ_id was not filled (it's True by default)

File: addons/hr_holidays/migrations/13.0.1.5/pre-migration.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_copies = {
+    'hr_leave': [
+        ('request_hour_from', None, None),
+        ('request_hour_to', None, None),
+    ],
+    'hr.leave.allocation': [
+        ('accrual', None, None),
```
Comment (MiquelRForgeFlow) [inline addons/hr_holidays/migrations/13.0.1.5/pre-migration.py]: in this in column_renames

File: addons/hr_holidays/migrations/13.0.1.5/pre-migration.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    ],
+    'hr.leave.type': [
+        ('categ_id', None, None),
```
Comment (MiquelRForgeFlow) [inline addons/hr_holidays/migrations/13.0.1.5/pre-migration.py]: in this in column_renames

File: addons/hr_holidays/migrations/13.0.1.5/pre-migration.py (hunk 3 above)
Comment (Yasaie) [reply addons/hr_holidays/migrations/13.0.1.5/pre-migration.py]: Thanks, done

File: addons/hr_holidays/migrations/13.0.1.5/pre-migration.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+_column_copies = {
+    'hr_leave': [
+        ('request_hour_from', None, None),
+        ('request_hour_to', None, None),
+    ],
+}
+
+_column_renames = {
+    'hr.leave.allocation': [
```
Comment (MiquelRForgeFlow) [inline addons/hr_holidays/migrations/13.0.1.5/pre-migration.py]:
```suggestion
    'hr_leave_allocation': [
```

File: addons/hr_holidays/migrations/13.0.1.5/pre-migration.py [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        ('accrual', None),
+    ],
+    'hr.leave.type': [
```
Comment (MiquelRForgeFlow) [inline addons/hr_holidays/migrations/13.0.1.5/pre-migration.py]:
```suggestion
    'hr_leave_type': [
```

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Made the final adjustments.

**Pull request merged**